src/services/drive/file/driveFile.service.ts

```diff
@@ -233,6 +236,38 @@ class DriveFileService {
     );
   }
 
+  public async getModifiedFiles({
+    limit = 50,
+    offset = 0,
+    updatedAt,
+    status,
+  }: {
+    limit?: number;
+    offset?: number;
+    updatedAt?: string;
+    status: 'ALL' | 'TRASHED' | 'REMOVED';
+  }): Promise<GetModifiedFiles[] | undefined> {
+    const query = `status=${status}&offset=${offset}&limit=${limit}${updatedAt && `&updatedAt=${updatedAt}`}`;
+    const newToken = await asyncStorageService.getItem(AsyncStorageKey.PhotosToken);
+
+    if (!newToken) return;
+
+    const headers = await getHeaders(newToken);
+
+    try {
+      const modifiedItems = await fetch(`${constants.DRIVE_NEW_API_URL}/files?${query}`, {
+        method: 'GET',
+        headers,
+      });
+
+      const parsedModifiedFiles = await modifiedItems.json();
+
+      return parsedModifiedFiles;
+    } catch (error) {
+      errorService.reportError(error);
```

Comment:
Propagate the error instead of catching it in the service, this way we can add context if we need to the reported error